extensions/amp-analytics/0.1/events.js

```diff
@@ -1384,15 +1411,25 @@ export class VideoEventTracker extends EventTracker {
         event.target,
         'No target specified by video session event.'
       );
-      targetReady.then((target) => {
-        if (!target.contains(el)) {
-          return;
-        }
-        const normalizedDetails = removeInternalVars(details);
-        listener(new AnalyticsEvent(target, normalizedType, normalizedDetails));
+
+      Promise.all(targetReadyPromises).then((targetReady) => {
+        targetReady.forEach((target) => {
+          if (!target[0].contains(el)) {
+            return;
+          }
+          const normalizedDetails = removeInternalVars(details);
+          listener(
+            new AnalyticsEvent(target[0], normalizedType, normalizedDetails)
+          );
+        });
       });
     });
   }
+
+  /**
+   * Assert that the selectors are all unique and are non-empty
+   * @param {!Array<string>|string} selectors
+   */
```

Comment:
I think we have to move these comments to above the `assertUniqueSelectors()` method?